server/scheduler123/src/main/java/org/sheduler/Shipper.java

```diff
@@ -0,0 +1,11 @@
+package org.sheduler;
+
+public class Shipper {
+    //An individual or company that has a load/freight that needs to be shipped.
+
+    private int loadingDocks;
+
+    public Shipper(int loadingDocks) {
+        this.loadingDocks = loadingDocks;
+    }
+}
```

Comment:
we also have to add the functionality of having freight to load, so far we only have unloading, which will depend on the carrier, while loading new freight will depend on the shipper